### ISSUE TITLE
Missing pipe on samples, can't get to the decoder

### ISSUE THREAD

**Issue by AinzAkaOver** [THUDM/CogVideo#613]
### System Info / 系統信息

Cuda 12.7
phyton  3.12
win 10 pro
3060 ti, 8g vram, 32gb ram, xeon e5 2678 v3

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [X] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

on the workflow I have a problem between samples and decoders, basically I'm using all the wokrflows with cogvideox sampler that  connects to the cogvideox decoder, this i sthe error
Prompt outputs failed validation

CogVideoDecode:
- Exception when validating inner node: tuple index out of range

basically if I right click and "fix node" the sampler just removes the cogvideo_pipe not that was connecting to the cogvideoencoder pipeline input and changes the name of the Encoder pipeline to VAE.

I tried to search and import the node, but from the wrapped I get the sampler with only 1 output while the workflow got 2

![Screenshot_19](https://github.com/user-attachments/assets/f3d4aae3-66fa-4b91-ae7f-06bbe5042a37)
 

### Expected behavior / 期待表现

connecting sampler with encoder

**Comment by AinzAkaOver** [comment]
then it's probably from ComfyUI-CogVideoXWrapper git, i'll get in touch with them
but do I have anything similar to replace those 2 nodes?
